core/symbol.rbs

```diff
@@ -152,21 +150,21 @@ class Symbol
   #
   # Related: String#<=>.
   #
-  def <=>: (Symbol other) -> Integer
-         | (untyped other) -> Integer?
+  def <=>: (Symbol object) -> (-1 | 0 | 1)
+         | (untyped) -> nil
```

Comment:
```suggestion
         | (untyped) -> Integer?
```

The overload may be applied to a `Symbol` object too.

Reply:
* #1875